notifications_utils/version.py

```diff
@@ -1 +1 @@
-__version__ = '15.0.0'
+__version__ = '15.0.1'
```

Comment:
minor thing, but this is a major version (new functionality) - so should be `15.1.0`